CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Libero
+We encourage contributions of any kind.  By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or ensure your employer has signed the CLA on your behalf.
+
+All Libero repositories will have further instructions in their individual `CONTRIBUTING.md` with more specific details.
+
+# Code Contributions
+The basic steps for code contributions are:
+1. Fork, then clone the repo
+2. Set up your machine and install dependencies
+3. Run all tests and ensure they pass
+4. Make your change, adding new tests
+5. Run all tests and ensure they pass
+6. Push to your fork and make a pull request
+
+We endeavour to respond to pull requests with an initial comment within 3 working days.
+
+You can increase the chance of getting your pull request accepted by:
+- Writing tests
+- Following the coding standards - (e.g. [PHP coding standards](https://github.com/libero/php-coding-standard))
+- Writing [a good commit message](https://chris.beams.io/posts/git-commit/)
+
+# Getting help
+- Report a bug or request a feature on [GitHub](https://github.com/libero/libero/issues/new/choose).
```

Comment:
Not relevant?

Reply:
I've removed this and spun it into a paragraph about larger contributions as @giorgiosironi suggested.